libc/test/src/math/DivTest.h

```diff
@@ -47,7 +47,7 @@ class DivTest : public LIBC_NAMESPACE::testing::FEnvSafeTest {
       InType x = InFPBits(v).get_val();
       InType y = InFPBits(w).get_val();
       mpfr::BinaryInput<InType> input{x, y};
-      EXPECT_MPFR_MATCH_ALL_ROUNDING(mpfr::Operation::Div, input, func(x, y),
+      ASSERT_MPFR_MATCH_ALL_ROUNDING(mpfr::Operation::Div, input, func(x, y),
                                      0.5);
```

Comment:
Isn't it better to have the test print all failures? My understanding was that `ASSERT_*` should be used if failure could prevent the rest of the test code from working as expected.

Reply:
These run ~100k times.  If many of them fail, it will be very noisy, and block or hang the terminal.  And they all need to pass anyway.

Reply:
Right, I think Ninja reads the whole output of each command it runs, then dumps it all at once, so that outputs don't get mixed up when running multiple commands at once. Printing more than just the first failure is useful when developing and fixing, but it's a very quick change, so I guess it can be done only locally when needed.